java/android/AmpliApp/app/src/main/java/com/amplitude/ampli/EnumCamelCase.java

```diff
@@ -0,0 +1,36 @@
+//
+// Ampli - A strong typed wrapper for your Analytics
+//
+// This file is generated by Amplitude.
+// To update run 'ampli pull java-ampli'
+//
+// Required dependencies: com.amplitude:android-sdk:2.34.1, com.squareup.okhttp3:okhttp:4.2.2
+// Tracking Plan Version: 0
+// Build: 1.0.0
+//
+// [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+//
+// [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/java-ampli)
+//
+package com.amplitude.ampli;
+
+import java.io.IOException;
+
+/**
+ * descriptionForEnumCamelCase
+ */
+public enum EnumCamelCase {
```

Comment:
Can you please namespace this like you did for Kotlin 🙏

Reply:
Done